src/AppBundle/Command/ListUsersCommand.php

```diff
@@ -0,0 +1,133 @@
+            ->addOption('send-to', null, InputOption::VALUE_OPTIONAL, 'If set, the result is sent to the given email address')
+        ;
+    }
+
+    /**
+     * This method is executed before the the execute() method. It's main purpose
+     * is to initialize the variables used in the rest of the command methods.
+     */
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->em = $this->getContainer()->get('doctrine')->getManager();
+    }
+
+    /**
+     * This method is executed after initialize(). It usually contains the logic
+     * to execute to complete this command task.
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $maxResults = $input->getOption('max-results');
+        // Use ->findBy() instead of ->findAll() to allow result sorting and limiting
+        $users = $this->em->getRepository('AppBundle:User')->findBy(array(), array('id' => 'DESC'), $maxResults);
+
+        // Doctrine query returns an array of objects and we need an array of plain arrays
+        $usersAsPlainArrays = array_map(function ($user) {
+            return array($user->getId(), $user->getUsername(), $user->getEmail(), implode(', ', $user->getRoles()));
+        }, $users);
+
+        // Instead of the common output type, this command uses the BufferedOutput
+        // type, which allows to get the contents before displaying them in the
+        // console. You probably should never use this kind of output, because it's
+        // only suited for very special use cases, like the one proposed by this command
+        $bufferedOutput = new BufferedOutput();
+
+        $table = new Table($bufferedOutput);
+        $table
+            ->setHeaders(array('ID', 'Username', 'Email', 'Roles'))
+            ->setRows($usersAsPlainArrays)
+        ;
+        $table->render();
+
+        // instead of displaying the table of users, store it in a variable
+        $tableContents = $bufferedOutput->fetch();
+
+        if (null !== $email = $input->getOption('send-to')) {
+            $this->sendReport($tableContents, $email);
+        }
+
+        $output->writeln($tableContents);
+    }
+
+    /**
+     * Sends the given $contents to the $recipient email address.
+     *
+     * @param string $contents
+     * @param string $recipient
+     */
+    private function sendReport($contents, $recipient)
+    {
+        // See http://symfony.com/doc/current/cookbook/email/email.html
+        $mailer = $this->getContainer()->get('mailer');
+
+        $message = $mailer->createMessage()
+            ->setSubject(sprintf('app:list-users report (%s)', date('Y-m-d H:i:s')))
+            ->setFrom($this->getContainer()->getParameter('app.notifications.email_sender'))
+            ->setTo($recipient)
+            ->setBody(sprintf('<pre>%s</pre>', $contents), 'text/html')
```

Comment:
you are sending a plain-text table inside a ``<pre>`` tag in HTML part. Why not sending a plain text part instead ?